BranchMonsterFactory/ViewControllers/MonsterCreatorViewController.m

```diff
@@ -51,9 +53,18 @@ - (void)viewDidLoad {
         [currView.layer setCornerRadius:currView.frame.size.width/2];
     }
     
+    //if it's null, create a new default monster
+    if (!self.editingMonster) {
+        self.editingMonster = [[BranchUniversalObject alloc] initWithCanonicalIdentifier:[NSString stringWithFormat:@"MONSTER_%u", arc4random_uniform(10000)]];
```

Comment:
following up, perhaps we could just call `[[BranchUniversalObject alloc] init];` here and set a canonical identifier before generating a link